src/validation/validation.cpp

```diff
@@ -2427,17 +2437,11 @@ bool ConnectBlock(const CBlock &block,
 
     // Discover how to handle this block
     bool canonical = enableCanonicalTxOrder.Value();
-    if (IsNov152018Scheduled())
+    // Always allow overwite of enableCanonicalTxOrder but for regtest
+    if (AreWeOnBCHChain() && IsNov152018Activated(chainparams.GetConsensus(), chainActive.Tip()) &&
+        !(chainparams.NetworkIDString() == "regtest"))
     {
-        // pindex-pprev != null because pindex is not genesis block (or fn would have returned above)
-        if (IsNov152018Enabled(chainparams.GetConsensus(), pindex->pprev))
-        {
-            canonical = true;
-        }
-        else
-        {
-            canonical = false;
-        }
+        canonical = true;
     }
```

Comment:
looks like you accidentally took out the   else{ canonical = false}  ?

Reply:
you are right again, even if we checkpointed the nov 15th fork we should give a chance to people to IBD with full checking enabled.